server-side/.env

```diff
@@ -1,5 +1,5 @@
-PORT=3000
+PORT=5000
 password=EuACQK5n92JI3vn4
-DB_URL="mongodb://127.0.0.1:27017/TimeOut"
+DB_URL='mongodb+srv://sh3000444:EuACQK5n92JI3vn4@cluster0.2ttcjcg.mongodb.net/?retryWrites=true&w=majority&appName=Cluster0'
```

Comment:
remove changing the mongo url
please add server-side/.env & package-lock.json to .gitignore file

Reply:
✅